driver/docker.go

```diff
@@ -82,6 +82,11 @@ func (c *DockerContainer) Command() string {
 	return c.cmdOverride
 }
 
+//GetPodID return pod-id assosicated with container.
```

Comment:
same typo as other drivers; and same comment about unused outside of CRI implementation